infra/blueprint-test/pkg/gcloud/gcloud.go

```diff
@@ -93,7 +94,10 @@ func RunCmdE(t testing.TB, cmd string, opts ...cmdOption) (string, error) {
 		t.Fatal(err)
 	}
 	// split command into args
-	args := strings.Fields(cmd)
+	args, err := shellwords.Parse(cmd)
```

Comment:
Docs seem to suggest env parsing is opt-in but wanted to confirm

Reply:
Yeah, for ENV parsing, my understanding (also looking at the source) is to instead use `shellwords.ParseWithEnvs()`.